src/api/ValuesApi.js

```diff
@@ -0,0 +1,114 @@
+import { polyfill } from 'es6-promise';
+import 'isomorphic-fetch';
+
+polyfill();
+
+export default {
+  name: 'ValuesApi',
+
+  getValues(name, params, options, searchState, qtext, facetObject) {
+    //get options
+    return fetch('/v1/config/query/all?format=json', {
```

Comment:
This should make a call to `/search/{type}/config`.. (see comments in grove-node PR)